### PR TITLE
level up tuts added

### LINKED ISSUE #1529: [Add] level up tuts
### Site URL

https://www.youtube.com/c/leveluptuts

### Description of the above provided site

2000+ free video tutorials created, recorded, edited, published and maintained by Scott Tolinski.

### Category

Youtube

### Sub Category

web development

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by S-ishita**
## Fixes Issue
closes #1529 

## Screenshots
Screenshot from 2023-08-10 09-46-36

### CODE REVIEW

File: database/youtube/web-development.json [hunk 1]
```diff
@@ -302,5 +302,13 @@
     "category": "youtube",
     "subcategory": "web-development",
     "language": "english"
+  },
+    {
+    "name": "Level Up Tuts",
+    "description": "2000+ free video tutorials created, recorded, edited, published and maintained by Scott Tolinski.",
```
Comment (CBID2) [inline database/youtube/web-development.json:308]:
```suggestion
    "description": "2000+ free video tutorials created, recorded, edited, published, and maintained by Scott Tolinski.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Found a typo

**Review by CBID2 — APPROVED** [review]
LGTM

**Review by k-deepak04 — DISMISSED** [review]

**Comment by k-deepak04** [comment]
@S-ishita please resolve the conflicts

**Comment by CBID2** [comment]
@k-deepak04 please review

**Pull request merged**